client/src/components/Grid/GridHistory.vue

```diff
@@ -66,6 +67,6 @@ withDefaults(defineProps<Props>(), {
         </BNav>
         <GridList v-if="activeList === 'my'" :grid-config="historiesGridConfig" embedded />
         <GridList v-else-if="activeList === 'shared'" :grid-config="historiesSharedGridConfig" embedded />
-        <GridList v-else :grid-config="historiesPublishedGridConfig" embedded />
+        <GridList v-else :grid-config="historiesPublishedGridConfig" :username-search="props.username" embedded />
```

Comment:
I think we usually don't include the props var, is there a convention around this ? If you want to be consistent, should `activeList` also be `props.activeList` ?

Reply:
I like being explicit about props but am not aware of a convention.